### PR TITLE
Feature/soft delete

### PR DESCRIPTION
**Pull request by celynolivia**
trx delete

### CODE REVIEW

File: database/database_admin.go [hunk 1]
```diff
@@ -433,7 +433,28 @@ func GetTotalFromLog(db *sql.DB, cons string) (totalVa int, totalAmount int, err
...
+	}
+
+	tLogs := models.LogAdmin{
+		AccNum: AccNum,
+		Action: constants.DELETE,
+	}
+
+	err = InsertLogAdminWithDbTransaction(tx, tLogs, username)
+	if err != nil {
+		fmt.Print(err)
```
Comment (vicimilenia) [inline database/database_admin.go]: please delete if not needed, thank you :)

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+JWT_SECRET=mysecretstring
+DATABASE_URL=postgres://postgres:password@127.0.0.1:5432/db_tsaving?sslmode=disable
+PORT=8000
```
Comment (polipopoliko) [inline .env]: remove this

Comment (vicimilenia) [inline .env]: please do not commit this thank you :)

**Review by vicimilenia — CHANGES_REQUESTED** [review]

File: database/database_admin.go (hunk 1 above)
Comment (polipopoliko) [inline database/database_admin.go]: i dont think we need this

**Review by polipopoliko — CHANGES_REQUESTED** [review]

**Review by polipopoliko — APPROVED** [review]

**Review by vicimilenia — APPROVED** [review]

**Pull request merged**